### PR TITLE
store/tikv: copy memory format utility to store/tikv

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

### What problem does this PR solve?
Former refactoring introduces dependency on `memory`.

Part of https://github.com/pingcap/tidb/issues/22513

### What is changed and how it works?
What's Changed:
- copy memory format utils into store/tikv

### Check List 
Tests 
- Unit test

### Release note 
- No release note

### CODE REVIEW

File: store/tikv/util/misc.go [hunk 1]
```diff
@@ -74,3 +76,60 @@ var SessionID = sessionIDCtxKey{}
...
+
+const (
+	byteSizeGB = int64(1 << 30)
+	byteSizeMB = int64(1 << 20)
+	byteSizeKB = int64(1 << 10)
+	byteSizeBB = int64(1)
+)
+
+// FormatBytes uses to format bytes, this function will prune precision before format bytes.
+func FormatBytes(numBytes int64) string {
```
Comment (AndreMouche) [inline store/tikv/util/misc.go:88]: Could we use this function in tidb because it is only used in log ... And it is just a proposal we can ignore it ..

**Review by AndreMouche — APPROVED** [review]
LGTM

**Review by sticnarf — APPROVED** [review]
LGTM

**Pull request merged**